js/app.jsx

```diff
@@ -92,6 +92,15 @@ function cleanJobPath(path) {
   return path;
 }
 
+function unique(array) {
```

Comment:
Yah I didn't care for it too much since it's deduping such a small array, but i suppose if this function ends up getting reused elsewhere it may be worth it to change it.